src/EnergyPlus/UnitarySystem.cc

```diff
@@ -1251,6 +1252,22 @@ namespace UnitarySystems {
             }
         }
 
+        // no water side economizer
+        this->runWaterSideEconomizer = false;
+        this->WaterSideEconomizerStatus = 0;
+        // re-set water-side economizer flags each time step
+        if (this->m_waterSideEconomizerFlag) {
+            // enable water-side economizer cooling
+            this->WaterSideEconomizerStatus = 1;
+            this->runWaterSideEconomizer = true;
+            // disable water-side economizer if entering fluid temp is > entering air temp minus user specified temp offset
+            if (state.dataLoopNodes->Node(this->CoolCoilFluidInletNode).Temp >
+                (state.dataLoopNodes->Node(this->AirInNode).Temp - this->m_minAirToWaterTempOffset)) {
+                this->runWaterSideEconomizer = false;
+                this->WaterSideEconomizerStatus = 0;
+            }
+        }
+
```

Comment:
This logic compares coil entering water (fluid) temperature, coil entering air temperature and user specified temperature offset to determine the coil control status.